shell/mixins/vee-validation.ts

```diff
@@ -0,0 +1,55 @@
+import Vue from 'vue';
+import { uniq } from 'lodash';
+import { veeTokenValidateUtil, VeeTokenRule } from '@shell/utils/vee-validators';
+
+interface Data {
+}
+
+export default Vue.extend<Data, any, any, any>({
+  props: {},
+
+  data() {
+    return { };
+  },
+
+  computed: {},
+
+  watch: {
+    resource: {
+
+      /**
+       * Can be triggered on parent component with emit()
+       */
+
+      async handler(neu) {
+        if (!this.$refs.validator) {
+          return;
+        }
+        const veeTokenRules: VeeTokenRule = this.$parent.veeTokenRules;
+
+        const rules = Object.values(veeTokenRules).map(({ id }) => id);
+
+        const unreportedRule = rules.find((id) => !Object.keys(this.$refs.validator.fields).includes(id));
```

Comment:
Could you add some comments about the `$refs`?
We may also need to catch edge cases.